mypy/nodes.py

```diff
@@ -101,7 +101,8 @@ def get_column(self) -> int:
 implicit_module_attrs = {'__name__': '__builtins__.str',
                          '__doc__': None,  # depends on Python version, see semanal.py
                          '__file__': '__builtins__.str',
-                         '__package__': '__builtins__.str'}
+                         '__package__': '__builtins__.str',
+                         '__path__': '__builtins__.list'}
```

Comment:
If you will fix this, you can then just take care to add `list.pyi` fixture to all tests that include `__init__.py[i]` files. I think there are not many of those.

Reply:
Are you saying that `__path__` shoun't be here because it exists only in `__init__`? So `__init__.py[i]` that uses `__path__` must include an fixture?

Reply:
Yes, please see the previous comments to this code. You already promised to fix it in https://github.com/python/mypy/pull/5212#discussion_r195136274 two months ago.

It is OK to just close this if you don't have time/desire to work on this. If you still want to do this, then please go through _all_ previous comments and fix mentioned issues.